app/assets/javascripts/matchdays.js

```diff
@@ -0,0 +1,25 @@
+function shiftWeek(event){
+
+
+  var viewDate = event.target.id;
+  var url = event.target.url;
+
+
+  $.ajax({
+    url: url,
+    type: 'GET',
+    dataType: 'json',
+    contentType: 'application/json',
+    data: JSON.stringify({
+      view_date: viewDate
+    })
+  })
+  .done(function(data){
+      $('#show-week-container').html(" ")
```

Comment:
I'm aware that i have to include the objects in  `data` in the .done somehow. My main problem is that I originally used render @match_days to render my match days. I can't figure out how to rerender these partials using the objects in `data` instead of the original @match_days.